cmd/res/updres.go

```diff
@@ -41,7 +42,7 @@ func init() {
 		"", "Name of the resource file")
 	UpdateCmd.Flags().StringVarP(&resType, "type", "p",
 		"", "Resource type. Valid types include java, js, jsc, properties, py, wsdl, xsd, or xsl.")
-	UpdateCmd.Flags().StringVarP(&resPath, "respath", "r",
+	UpdateCmd.Flags().StringVarP(&resPath, "respath", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `respath` is breaking change right?